fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerAppointments.js

```diff
@@ -172,17 +179,44 @@ function AdminAccounts() {
     };
 
     const updateAppointmentStatus = (appointmentId, status) => {
-        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments/${appointmentId}/${status}`, {
+        fetch(`http://localhost:8080/api/v1/appointments/${appointmentId}/cancelled`, {
```

Comment:
You forgot to chanhe the url to ${process.env.REACT_APP_BASE_URL}

Reply:
Oups thanks for reminding me i will change it.